flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/utils/IntervalJoinUtil.scala

```diff
@@ -450,15 +452,26 @@ object IntervalJoinUtil {
    *         else false.
    */
   def satisfyIntervalJoin(join: FlinkLogicalJoin): Boolean = {
+    satisfyIntervalJoin(join, join.getLeft, join.getRight)
```

Comment:
It seems that only one place holds this old function, why not delete it and use the new function format everywhere?

Reply:
If no new left node and right node, there is no need to pass the original left and right again because we could get left/right from join node.
It's a shortcut, besides, does not introduce unnessassary change.